pkg/backend/filestate/state.go

```diff
@@ -49,6 +49,19 @@ const DisableCheckpointBackupsEnvVar = "PULUMI_DISABLE_CHECKPOINT_BACKUPS"
 // be used as a last resort when a command absolutely must be run.
 var DisableIntegrityChecking bool
 
+type cloudQuery struct {
```

Comment:
```suggestion
type localQuery struct {
```

Reply:
Heh. This actually is meant to reflect that it is part of the `CloudQuery` SDK! But, ok. `localQuery` it is.